packages/ihe-gateway/server/CodeTemplates/XCA Library - Outbound/Get XCA ITI-38 response template/Get XCA ITI-38 response template.js

```diff
@@ -8,7 +8,10 @@
 function getXCA38ResponseTemplate(request, operationOutcome) {
 	
 	var result = request;
-	result.responseTimestamp = DateUtil.getCurrentDate("yyyy-MM-dd'T'hh:mm:ss.sssZ");
+  var requestTime = channelMap.get("REQUEST_TIME");
+  var responseTime = DateUtil.getCurrentDate("yyyy-MM-dd'T'hh:mm:ss.SSSZ");
```

Comment:
Note sss was changed to SSS it didnt actually get the milliseconds unless capitalized